packages/dev/core/src/Debug/debugLayer.ts

```diff
@@ -261,18 +264,39 @@ export class DebugLayer {
         return this._onSelectionChangedObservable;
     }
 
+    /**
+     * the default configuration of the inspector at the class level
+     */
+    public static Config: IInspectorOptions = {
+        overlay: false,
+        showExplorer: true,
+        showInspector: true,
+        embedMode: false,
+        handleResize: true,
+        enablePopup: true,
+    };
+
+    /**
+     * the default configuration of the inspector
+     * DebugLayer.Config is inherited when the instance is created
+     * After it is created, changes to DebugLayer.Config are not synchronized to the instance configuration
+     */
+    public config: IInspectorOptions = { ...DebugLayer.Config };

```

Comment:
Small ask regarding consistency - it would be great if you use the spread operator in the constructor to assign both the passed config and `DebugLayer.Config`  instead of assigning it once here (using spread) and then using Object.assign in the constructor.

Reply:
Also I am honestly not sure we need a per instance cache as well as a static cache. We could only keep the static one @GuoBinyong